.github/workflows/macos-arm64-release.yml

```diff
@@ -0,0 +1,40 @@
+name: MacOs arm64 release
+
+on:
+  push:
+    tags:
+      - '*'
+
+env:
+  RELEASE_ARCHIVE_NAME: macOS-arm64-release
+
+jobs:
+  build-macos-arm64:
+    runs-on: macos-14
```

Comment:
I found that macos-latest can provide runner with less version, so to build on arm I use certain. Maybe there is some convenient way to specify arch? I didn't found explicit way in workflow docs. 
It must be handle in future: using cross-compilation (according docs of pjsip it is experimental), or just keep hardcoded version and increase it if needed